src/main/java/net/mcreator/ui/dialogs/MCItemSelectorDialog.java

```diff
@@ -168,18 +169,31 @@ public MCItemSelectorDialog(MCreator mcreator, MCItem.ListProvider blocksConsume
 		ComponentUtils.deriveFont(jtf, 15);
 		ComponentUtils.deriveFont(filterField, 15);
 
+		JComponent top;
 		JButton all = L10N.button("dialog.item_selector.all");
 		all.addActionListener(event -> filterField.setText(""));
 		JButton blocks = L10N.button("dialog.item_selector.blocks");
 		blocks.addActionListener(event -> filterField.setText("block"));
 		JButton items = L10N.button("dialog.item_selector.items");
 		items.addActionListener(event -> filterField.setText("item"));
 		JButton mods = L10N.button("dialog.item_selector.custom_elements");
-		mods.addActionListener(event -> filterField.setText("mcreator"));
-
-		JComponent top = PanelUtils.join(FlowLayout.LEFT, L10N.label("dialog.item_selector.name"), jtf,
-				L10N.label("dialog.item_selector.display_filter"), filterField, new JLabel(""), all, blocks, items,
-				mods);
+		mods.addActionListener(event -> filterField.setText("custom"));
+
+		List<MCItem> itemsList = blocksConsumer.provide(mcreator.getWorkspace());
+		boolean hasPotions = itemsList.equals(ElementUtil.loadBlocksAndItemsAndPotions(mcreator.getWorkspace())) ||
```

Comment:
You should make this the same way as it is done with tags with supportTags parameter, so we don't need to run list comparison here as I would avoid this if possible.